webservice-session/session-leon/src/main.py

```diff
@@ -50,35 +34,58 @@ def validate_password():
     return Response("valid", status=200)
   return Response("invalid", status=200)
 
-@app.route('/items',methods = ['PUT'])
-def update_item():
-    request_data = request.get_json()
-    index = request_data["id"]
-    item = request_data['item']
-    status = request_data['status']
-    reminder = request_data['reminder']
-    added_item = item_actions.update_item(index, item, status ,reminder)
-    return   added_item
 
+@app.route('/todos/<id>', methods = ['GET'])
+def get_todos(id):
+  res = Todos_api.get_todos(id)
+  return res
+
+
+
+
+
+#api test done / unit test done
```

Comment:
comments can be ignored.
All the other comments in the file can also be ignored.